spec/index.html

```diff
@@ -409,6 +410,23 @@ <h3>rdf:type</h3>
         href="#ch_class"><code>rdfs:Class</code></a>.</p>
   </section>
 
+  <section id="ch_reifies">
+    <h3>rdf:reifies</h3>
+    <p><code>rdf:reifies</code> is an instance of <a href="#ch_property"><code>rdf:Property</code></a>
+      that is used to associate a resource (reifier) with a triple (proposition).</p>
+    <p>A triple of the form:</p>
+    <blockquote> <code>R rdf:reifies &lt;&lt;S P O&gt;&gt;</code> </blockquote>
+    <p>states that R is an instance of <a href="#ch_resource"><code>rdfs:Resource</code></a> 
+      and reifies the triple term <code>&lt;&lt;S P O&gt;&gt;</code>, where S, P, and O represent 
+      the subject, predicate, and object of the reified triple, respectively. The object 
```

Comment:
I believe that the R, S, P, and O variables should be tagged to set them apart from the surrounding text, whether `<em>`, `<code>`, or otherwise. (And that similar tagging should be applied to other places where there's a `<code>`-tagged sequence that is then deconstructed.) I'm applying `<em>` here, as that is what was applied in the ill-fated [w3c/rdf-semantics/#52](https://github.com/w3c/rdf-semantics/pull/52)
```suggestion
    <p>states that <em>R</em> is an instance of <a href="#ch_resource"><code>rdfs:Resource</code></a> 
      and reifies the triple term <code>&lt;&lt;S P O&gt;&gt;</code>, where <em>S</em>, <em>P</em>, and <em>O</em> represent 
      the subject, predicate, and object of the reified triple, respectively. The object 
```

Reply:
I believe you can use `|R|` (or `<var>R</var>`) for variables, and if in an element tagged "algorithm", it will highlight all the places the variable is used (within that element), at least in the Editor's Draft. The main thing is to be consistent